client/my-sites/backup/status/hooks.js

```diff
@@ -142,7 +142,9 @@ export const useDailyBackupStatus = ( siteId, selectedDate ) => {
 
 	return {
 		isLoading:
-			lastBackupBeforeDate.isLoading || lastAttemptOnDate.isLoading || backupDeltas.isLoading,
+			lastBackupBeforeDate.isLoading ||
+			lastAttemptOnDate.isLoading ||
```

Comment:
```suggestion
			lastBackupBeforeDate.isInitialLoading ||
			lastAttemptOnDate.isInitialLoading ||
```

I think these are also affected and need change since in the end, they are calling `useRewindableActivityLogQuery` in `useLatestBackupAttempt` hook. In fact, there are some places in the codebase that uses `useRewindableActivityLogQuery` (and `isLoading`) that need fixing :(

Reply:
I'd rather keep this fix as small as possible. We can tackle these ones in a separate PR maybe and perform more broader testing then. Sounds good?

Reply:
Yep